### PR TITLE
CORE-18641 Initial work for sandbox testing in isolation

### PR DESCRIPTION
**Pull request by bpaunescu**
- Added stress tests for Flow and Persistence sandboxes
- Added some test metrics
- Added utilities to facilitate metrics data consumption using Prometheus and Grafana

### CODE REVIEW

File: applications/tools/metrics-webserver/build.gradle [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    api "info.picocli:picocli:$picocliVersion"
+}
+
+tasks.named('jar', Jar) {
+    duplicatesStrategy = DuplicatesStrategy.INCLUDE
+
+    from (sourceSets.main.output)
+    from { configurations.runtimeClasspath.collect { it.isDirectory() ? it : zipTree(it) } }
+
+        manifest {
```
Comment (ronanbrowne) [inline applications/tools/metrics-webserver/build.gradle:30]: Looks like the formatting is off there?

File: applications/tools/metrics-webserver/build.gradle [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+dependencies {
+    compileOnly 'org.jetbrains.kotlin:kotlin-stdlib'
+
+    implementation libs.javalin
+    implementation libs.slf4j.simple
+
+    api "info.picocli:picocli:$picocliVersion"
+}
+
+tasks.named('jar', Jar) {
```
Comment (ronanbrowne) [inline applications/tools/metrics-webserver/build.gradle:22]: Is there any expectation for this Jar to be published externally to Maven Central as part of our release process or is the presumption a user will always build it if needed as described in readme.md

**Review by ronanbrowne — CHANGES_REQUESTED** [review]
Few queries on `applications/tools/metrics-webserver/build.gradle` i'll leave the *kt code to a platform engineer to review.

File: applications/tools/metrics-webserver/build.gradle (hunk 2 above)
Comment (bpaunescu) [reply applications/tools/metrics-webserver/build.gradle:22]: There is no current plan to publish this anywhere. It's sufficient to build a jar locally or run it directly from Intellij.

File: applications/tools/metrics-webserver/build.gradle (hunk 1 above)
Comment (bpaunescu) [reply applications/tools/metrics-webserver/build.gradle:30]: Attempted to improve it.

**Review by ronanbrowne — APPROVED** [review]
LGTM re my comments

**Review by kyriathar — APPROVED** [review]
OK

**Pull request merged**